app/styles/_profile.scss

```diff
@@ -1,5 +1,7 @@
 .hero-profile {
   max-width: $profile-bar-width;
+  min-width: calc(#{$profile-bar-width} / 1.6);
```

Comment:
Is `calc` working correctly in browsers now? I mean, without killing all the performance? :older_man: